File: 프로그래머스/0/181873. 특정한 문자를 대문자로 바꾸기/특정한 문자를 대문자로 바꾸기.cpp
#include <string>
#include <vector>
#include <cctype>
#include <iostream>
using namespace std;

string solution(string my_string, string alp) {
    string answer = "";
    
    for(auto& i: my_string){
            if(i== alp[0]){
                //i를 대문자로 바꾸는함수
                i = toupper(i);
            }
    }
    cout << my_string;
    
    return my_string;
}

File: 프로그래머스/0/181873. 특정한 문자를 대문자로 바꾸기/특정한 문자를 대문자로 바꾸기.java
class Solution {
    public String solution(String my_string, String alp) {
        StringBuilder answer = new StringBuilder();
        
        for(int i=0; i<my_string.length(); i++){
            
            if(String.valueOf(my_string.charAt(i)).equals(alp)){
                answer.append(String.valueOf(my_string.charAt(i)).toUpperCase());
            } else{
                answer.append(my_string.charAt(i));
            }
            
        }
        
        
        return answer.toString();
    }
}